catt/controllers.py

```diff
@@ -184,25 +189,39 @@ def clear(self):
 
 class Cache(CattStore):
     def __init__(self):
-        cache_path = Path(tempfile.gettempdir(), "catt_cache", "chromecast_hosts")
+        cache_path = Path(tempfile.gettempdir(), "catt_%s_cache" % CATT_VERSION, "chromecast_hosts")
```

Comment:
Can you sha1 the version here and get the first 8 chars or so? It's not very likely, but I'd like to guard against a version containing slashes like `0.4.3-pre1/amd` or whatever.

Reply:
Yep

Reply:
Done